package.json

```
@@ -1,9 +1,12 @@
 {
-  "name": "@folder/readdir",
+  "name": "readdir",
```

Was this supposed to change?

```suggestion
  "name": "@folder/readdir",
```

---

Good catch. Not sure how this got changed.